src/Ghcid.hs

```diff
@@ -308,7 +308,10 @@ runGhcid session waiter termSize termOutput opts@Options{..} = do
         putStrLn $ "\nNo files loaded, GHCi is not working properly.\nCommand: " ++ command
         exitFailure
 
-    restart <- return $ nubOrd $ restart ++ [x | LoadConfig x <- messages]
+    let stripTrailingSep = reverse . dropWhile (`elem` pathSeparators) . reverse
+    curDir <- getCurrentDirectory
+    let absRestart = map (stripTrailingSep . (curDir </>)) restart
```

Comment:
Is this whole purpose of this bit just https://hackage.haskell.org/package/directory-1.3.6.0/docs/System-Directory.html#v:makeAbsolute ?